content/20.treatments.md

```diff
@@ -147,19 +147,47 @@ Summarize the state of the antiviral approach.
 
 #### Protease Inhibitors
 
-Why it may be useful
+The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.
 
-##### Protease Inhibitor 1
+##### N3
+N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcriptipn.
 
 ###### Anticipated Mechanism
+N3 inhibits Mpro through binding to its substrate pocket.
 
 ###### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+N3 was first designed computationally [@doi:10.1371/journal.pbio.0030428] to bind in the substrate binding pocket of SARS-CoV and MERS-CoV Mpro proteases [@doi:/10.1007/s13238-013-2841-3]. 
```

Comment:
```suggestion
N3 was first designed computationally [@doi:10.1371/journal.pbio.0030324] to bind in the substrate binding pocket of the Mpro protease of SARS-like coronaviruses [@doi:10.1007/s13238-013-2841-3]. 
```

Great, thanks for finding these.  I updated the first one.  I believe that `pbio.0030428` points to the correction, not the original article.

I reworded the second part because that short article was mostly about HCoV-EMC, and I didn't see it reference MERS.